ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -301,24 +305,50 @@ const PANELS = PANEL_DETAILS.map(panelDetails)
 const STAFF_PANELS = STAFF_PANEL_DETAILS.map(panelDetails)
 
 class VariantSearchFormContent extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { activeIndex: [] }
+  }
+
+  expandAll = () => {
+    this.setState({ activeIndex: [0, 1, 2, 3, 4, 5] })
```

Comment:
rather than hardcoding this as a list of integers that will have to be maintained if we ever change the number of panels, I would compute this based on the length of `PANELS`. A neat way to do this in es6 is `[...PANELS.keys()]`